docs/content/installation.md

```diff
@@ -253,7 +259,7 @@ First of all we need to tell Wakame-vdc how we are storing these *backup objects
 
 ```bash
 backupstorage add \
-  --uuid bkst-local \
+  --uuid bkst-demo1 \
```

Comment:
I only just noticed this change. This will break instance backups since 'bkst-local' is set as the storage backupstorage to use when backing up instances. https://github.com/axsh/wakame-vdc/blob/develop/dcmgr/config/dcmgr.conf.example